logit.m

```diff
@@ -1,4 +1,6 @@
 function L = logit(p)
```

Comment:
that's a matlab issue, but can you explain what the type of p is - e.g. Array{Float64} (bonus)?